CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# UNRELEASED
+
+- Dependency Updates:
```

Comment:
we should probably clarify this so that an outside dev would know which are dev dependency updates and which will impact them (node)

Reply:
Good idea. We'll add that to each of these PRs

Reply:
@hollabaq86 added these changes